internal/common/common.go

```diff
@@ -93,6 +94,33 @@ const NMDebugModeConf = `
 domains=ALL:DEBUG
 `
 
+func GetIgnoredValidations(validationsJSON string, clusterID string) ([]string, bool) {
```

Comment:
Something i don't get here, it is actually the same as DeserializeJSONList, no? Maybe i somehow confused you